source/common/tracing/zipkin_tracer_impl.cc

```diff
@@ -0,0 +1,201 @@
+#include "common/tracing/zipkin_tracer_impl.h"
```

Comment:
In the very initial discussions we had with @RomanDzhabarov, he suggested that the Zipkin-related code, similar to the LightStep code, should be under the Tracing namespace. In fact, lightstep_tracer_impl.cc and zipkin_tracer_impl.cc are quite similar. I see those files as some sort of conceptual template for the implementation of other tracing drivers.
Do you feel strongly about moving that to the zipkin directory and to the Zipkin namespace?